src/lib/mod.rs

```diff
@@ -49,58 +54,72 @@ impl Transaction {
 
 impl TransactionType {
     pub fn compute_and_hash(&self) -> Result<String> {
-        let mut hasher = Sha256::new();
-
         match self {
             TransactionType::FunctionExecution {
-                program: program_str,
+                program: _,
                 function,
-                program_name: _,
-                enable_trace: execute_trace,
+                program_name,
+                enable_trace: _
```

Comment:
Is there no way to enable/disable the trace?

Reply:
Removed because `starknet-rs` does not support giving the trace